runtime/setup.py

```diff
@@ -568,7 +568,9 @@ def populate_built_package(abs_dir):
                 "iree-run-trace*",
                 "iree-benchmark-module*",
                 "iree-benchmark-trace*",
+                # These utilities are invariant wrt tracing and are only build for the default runtime.
                 "iree-dump-module*",
+                "iree-dump-parameters*",
                 "iree-cpuinfo*",
```

Comment:
nit: grammar ("are only built" or "only build", not "are only build")

```suggestion
                # These utilities are invariant wrt tracing and are only built for the default runtime.
                "iree-dump-module*",
                "iree-dump-parameters*",
                "iree-cpuinfo*",
```